src/components/ui.js

```diff
@@ -0,0 +1,38 @@
+
+import Weather from './request';
+
+const searchbtn = document.querySelector('#change-btn');
+
+
+const weather = new Weather(document.querySelector('#city').value, document.querySelector('#country').value);
+
+const populateUi = () => {
+  const getReport = () => {
+    weather.getWeather()
+      .then(result => {
+        document.querySelector('#location').textContent = `${result[1].country}`;
+        document.querySelector('#description').textContent = `The weather will be ${result[0][0].description}`;
+        document.querySelector('#temp').textContent = `Temperature is ${result[2].temp}`;
+      })
+      .catch(() => {
+        const errorMessage = document.querySelector('#error');
+        errorMessage.setAttribute('class', 'alert alert-warning');
+        errorMessage.setAttribute('role', 'alert');
+        errorMessage.textContent = 'PLEASE SEARCH FOR A VALID CITY AND COUNTRY';
```

Comment:
## Add the functionality to view weather changes based on appearance.
- [ ] As part of the mandatory requirements of this project. You are expected to add the functionality of having the weather results getting displayed and them having changes based on the appearance based on the received data as explained in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/appearance_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app).

## Add the toggle functionality
- [ ] the project needs to have a toggle control to determine if the result will be shown in Celsius or Fahrenheit *(There are two valid solutions: you can make two different requests for the Celsius data and the Fahrenheit data OR make only one request and use math to convert it to the other unit)* as explained in this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/weather_app/search_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/weather-app).

Currently, this is what we currently have 👇 
![weather-app](https://user-images.githubusercontent.com/22345961/109196961-728fb400-77ad-11eb-9b31-bc5a0087fc5a.png)